### PR TITLE
Devenv: Add `undev` command to the ./setup scrip of devenv

### PR DESCRIPTION
**Pull request by eleijonmarck**
**What is this feature?**

To remove the devenv setup, setup by initiating the ./setup.sh script.

### CODE REVIEW

File: devenv/setup.sh [hunk 1]
```diff
@@ -84,6 +112,8 @@ main() {
 		bulkDashboard
 	elif [[ $cmd == "bulk-folders" ]]; then
 		bulkFolders "$arg1"
+	elif [[ $cmd == "undev" ]]; then
```
Comment (gamab) [inline devenv/setup.sh:115]: I think we should update the usage function as well :)

File: devenv/setup.sh [hunk 2]
```diff
@@ -59,6 +59,34 @@ devDatasources() {
 		ln -s -f ../../../devenv/datasources.yaml ../conf/provisioning/datasources/dev.yaml
 }
 
+undev() {
+    echo -e "\xE2\x9C\x94 Reverting all dev provisioning"
+
+    # Removing generated dashboard files for bulkDashboard
```
Comment (gamab) [inline devenv/setup.sh:65]: I think we are missing bulk-folders as well

File: devenv/setup.sh [hunk 3]
```diff
@@ -59,6 +59,34 @@ devDatasources() {
...
+    COUNTER=0
+    MAX=400
+    while [ $COUNTER -lt $MAX ]; do
+        rm -f "bulk-dashboards/dashboard${COUNTER}.json"
+        let COUNTER=COUNTER+1
+    done
```
Comment (gamab) [inline devenv/setup.sh:71]:
```suggestion
    done
    echo -e "    \xE2\x9C\x94 Reverting bulk-dashboards provisioning"
```

File: devenv/setup.sh [hunk 4]
```diff
@@ -59,6 +59,34 @@ devDatasources() {
...
+
+    # Removing generated dashboard and datasource files for bulkAlertingDashboard
+    COUNTER=0
+    MAX=1000
+    while [ $COUNTER -lt $MAX ]; do
+        rm -f "bulk_alerting_dashboards/alerting_dashboard${COUNTER}.json"
+        let COUNTER=COUNTER+1
+    done
```
Comment (gamab) [inline devenv/setup.sh:79]:
```suggestion
    done
    echo -e "    \xE2\x9C\x94 Reverting bulk-alerting-dashboards provisioning"
```

File: devenv/setup.sh [hunk 5]
```diff
@@ -59,6 +59,34 @@ devDatasources() {
...
+    rm -f "bulk_alerting_dashboards/bulk_alerting_datasources.yaml"
```
Comment (gamab) [inline devenv/setup.sh:80]: I think we can do simpler. Also I think you might have missed some files :slightly_smiling_face: 
```suggestion
    # Removing generated dashboard files from bulk-dashboards
    rm -f bulk-dashboards/dashboard*.json
    rm -f bulk-dashboards/bulk-dashboards.yaml
    echo -e "    \xE2\x9C\x94 Reverting bulk-dashboards provisioning"

    # Removing generated folders from bulk-folders
    rm -rf bulk-folders/Bulk\ Folder*
    rm -f bulk-folders/bulk-folders.yaml
    echo -e "    \xE2\x9C\x94 Reverting bulk-folders provisioning"


    # Removing generated dashboard and datasource files for bulk-alerting-dashboards
    rm -f bulk_alerting_dashboards/alerting_dashboard*.json
    rm -f "bulk_alerting_dashboards/bulk_alerting_dashboards.yaml"
    rm -f "bulk_alerting_dashboards/bulk_alerting_datasources.yaml"
    echo -e "    \xE2\x9C\x94 Reverting bulk-alerting-dashboards provisioning"
```

**Review by gamab — COMMENTED** [review]
Nice thanks for doing this. 
I think we can simplify a bit the function (not to loop). See my suggestion :slightly_smiling_face:

File: devenv/setup.sh (hunk 5 above)
Comment (eleijonmarck) [reply devenv/setup.sh:80]: @gamab did you try this out?

Comment (gamab) [reply devenv/setup.sh:80]: Yes locally on my linux machine :)

File: devenv/setup.sh (hunk 1 above)
Comment (gamab) [reply devenv/setup.sh:115]: You are adding a new command to the script, but did not mention that command is available in the `Usage` function a few lines above.

**Review by gamab — APPROVED** [review]

**Pull request merged**